.github/actions/setup-deps/action.yml

```diff
@@ -51,30 +63,24 @@ runs:
             helm plugin install https://github.com/chartmuseum/helm-push
         fi
     - name: Install Go
+      if: ${{ inputs.install-golang == 'true' }}
       uses: actions/setup-go@v5
       env:
         DEBIAN_FRONTEND: noninteractive
       with:
         go-version-file: go.mod
-    - name: Get Required Python Version
-      id: get-python-version
-      uses: mikefarah/yq@master
-      env:
-        DEBIAN_FRONTEND: noninteractive
-      with:
-        cmd: >-
-          yq
-          '.project.requires-python | capture("(?<version>3.[0-9]+(\.[0-9]+)?)").version'
-          pyproject.toml
-    - name: Install Py (${{ steps.get-python-version.outputs.result }})
-      uses: actions/setup-python@v5
+    - name: Install 'uv'
+      shell: bash
+      if: ${{ inputs.install-python == 'true' }}
       env:
         DEBIAN_FRONTEND: noninteractive
-      with:
-        python-version: ${{ steps.get-python-version.outputs.result }}
+      run: |
+        curl -LsSf https://astral.sh/uv/install.sh | bash
     - name: Install Python Requirements
       shell: bash
+      if: ${{ inputs.install-python == 'true' }}
       env:
         DEBIAN_FRONTEND: noninteractive
       run: |
-        python -m pip install -U awscli packaging
+        uv venv
+        uv --no-config pip install -U awscli
```

Comment:
I think `awscli` is in the dev dependencies now, no?

(I also expect we'll likely get rid of it – I _think_ it's only ALabs-specific stuff.)

Reply:
I did move everything else that _used_ to be installed in that step into the dev deps, but nothing in the Python packages explicitly used anything from `awscli`, so I figured maybe it was a CI-specific thing and just left it.

We can absolutely sunset it / remove it entirely though, if it's an ALabs-exclusive